README.md

```diff
@@ -29,5 +29,11 @@ instead.
 
 ## Developing
 
-- If you are adding new Concourse flags to one of the job specs you must run `scripts/generate-job-templates` to add the new flags to the job templates.
-- When adding a new Concourse flag, don't define a `default` value that mirrors a default set by the Concourse binary. Instead, you may use the `example`. This prevents the possibility of drift if the Concourse binary default value changes.
+If you are adding new Concourse flags to one of the job specs you must run `scripts/generate-job-templates` to add the new flags to the job templates.
+
+**Default values**
+
+For Concourse deployments on BOSH made using the default values of the release, this prevents the behaviour drifting from that of the binary in case the binary's default values change.
+
+Note that the comment stating the binary's default can still become out of sync when such an update happens. The current solution is a suboptimal one. It may be improved in the future by generating a list of the default values here from the binary.
+
```

Comment:
Hey,
Found this section a bit hard to follow. If the older statement isn't clear, should we explicitly add a reference to an example like [this](https://github.com/concourse/concourse-bosh-release/blob/master/jobs/worker/spec#L173-L178) ? Or was the intention to call out that the current existing of default values in the release is sub-optimal and if we are making changes in an area, we should attempt to clean them up ?

Reply:
Oh good catch! So I'm going to talk about two things now - 1) the `example:` field and 2) the "example" of mentioning a default. 

1. I was going off the `concourse-chart` and forgot we use an `example` field in the BOSH release to mention the default. However, I'm realizing putting something is an `example` doesn't really communicate to operators that if they don't put in a value this is what Concourse will default to.

2. Putting in an example of how to mention default is a good suggestion. I'll try to make the sentences clear and put in an example like the following:

```
containerd.max_containers:
    env: CONCOURSE_CONTAINERD_MAX_CONTAINERS
    description: |
      Maximum container capacity. 0 means no limit. Defaults to 250.
```